CHEWBBACA/CHEWBBACA_NS/send2NS_refactored.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep 20 14:45:07 2019
+
+@author: pcerqueira
+"""
+
+import os
+import csv
+import json
+import time
+import requests
+from collections import defaultdict
+#import extra_scripts.utils as ut
+
+def process_genome(genome, schemaURI, profile, listHeaders, headers_post):
+
```

Comment:
this functions needs a little cleanup, and some comments as it's not very easy to understand